assets/dev/js/editor/utils/input-utils.js

```diff
@@ -0,0 +1,11 @@
+export default class InputUtils {
+	static isPaste( inputEvent ) {
+		const PASTE_EVENT = 'insertFromPaste';
+		const KEY_V = 'KeyV';
+
+		const originalEventPaste = PASTE_EVENT === inputEvent.originalEvent.inputType;
+		const ctrlPlusV = inputEvent.code === KEY_V && inputEvent.ctrlKey;
```

Comment:
Did you check for  `shift + insert`? And what about Mac's `cmd + v`?

Reply:
`metaKey`